src/test/ui/parser/float-literals.rs

```diff
@@ -0,0 +1,9 @@
+// compile-pass
```

Comment:
```suggestion
// build-pass
```

My bad, this is the correct tag.

Reply:
This should fix it. I had errors with other tests (function-names) due to weird dependencies, so I couldn't run the entire suite, but `./x.py test --stage 1 src/test/ui/parser` passed.